tools/project_manager/edit_common.inc

```diff
@@ -316,12 +316,31 @@ function maybe_echo_placeholder_option($initial)
 
 // XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 
-function proj_comments_field($comments, $field_name, $comment_format)
+function proj_comments_format($comment_format)
+{
+    $comment_formant_list = array(
```

Comment:
s/comment_formant_list/comment_format_list/ (also where it is used in the foreach below)